### PR TITLE
Added Next and Prev, Autoplay next song with shuffle functionality

### PR DESCRIPTION
**Pull request by rahulpoonia29**
As mentioned in issue https://github.com/Satyam1923/Spring-Music-Player/issues/25

### CODE REVIEW

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React, { useState, useEffect } from "react";
+import React, { useState, useEffect, useRef } from "react";
 import AudioPlayer from "react-h5-audio-player";
```
Comment (Satyam1923) [inline frontend/src/App.jsx:2]: implement our own player

File: frontend/src/App.jsx [hunk 2]
```diff
@@ -64,22 +72,49 @@ const App = () => {
...
+                            onEnded={nextPlay}
+                        />
+                        <div className="flex justify-between p-3 items-center">
+                            <button
+                                className="bg-gray-400 p-2 rounded-lg"
+                                onClick={previousPlay}
+                            >
+                                Previous
+                            </button>
+                            <button
```
Comment (Satyam1923) [inline frontend/src/App.jsx:96]: these button should be inside player

Comment (rahulpoonia29) [reply frontend/src/App.jsx:96]: The player is imported.

**Comment by Satyam1923** [comment]
behind commits.

**Pull request closed without merging**